cypress-tests/cypress/e2e/PaymentUtils/Nexixpay.js

```diff
@@ -6,43 +16,119 @@ const successfulThreeDSTestCardDetails = {
   card_cvc: "396",
 };
 
+const customerAcceptance = {
+  acceptance_type: "offline",
+  accepted_at: "1963-05-03T04:07:52.723Z",
+  online: {
+    ip_address: "125.0.0.1",
+    user_agent: "amet irure esse",
+  },
+};
+
+const multiUseMandateData = {
+  customer_acceptance: customerAcceptance,
+  mandate_type: {
+    multi_use: {
+      amount: 8000,
+      currency: "EUR",
+    },
+  },
+};
+
+const singleUseMandateData = {
+  customer_acceptance: customerAcceptance,
+  mandate_type: {
+    multi_use: {
+      amount: 8000,
+      currency: "EUR",
+    },
+  },
+};
+
+const payment_method_data_no3ds = {
+  card: {
+    last4: "1111",
+    card_type: null,
+    card_network: null,
+    card_issuer: null,
+    card_issuing_country: null,
+    card_isin: "411111",
+    card_extended_bin: null,
+    card_exp_month: "08",
+    card_exp_year: "25",
```

Comment:
increase this year to a later date. august 2025 will come sooner than you might expect.